Amazon.IonObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -15,12 +15,6 @@ public class IonObjectSerializerTest
     {
         IonSerializer defaultSerializer = new IonSerializer();
         private IValueFactory valueFactory = new ValueFactory();
-
-        [TestMethod]
-        public void SerializesAndDeserializesObjects()
```

Comment:
I think reinstate this test, and, rather than using `honda` in your test, make a new test object.

Reply:
yeah created a new test object.